src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -20,20 +20,28 @@
 namespace Confluent.Kafka
 {
     /// <summary>
-    ///     Represents the low and high watermark offsets of
-    ///     a Kafka topic/partition.
+    ///     Represents the low and high watermark offsets of a Kafka 
+    ///     topic/partition.
     /// </summary>
+    /// <remarks>
+    ///     You can identify a partition that has not yet been written
+    ///     to by checking if the high watermark equals 0.
+    /// </remarks>
     public class WatermarkOffsets
     {
         /// <summary>
         ///     Initializes a new instance of the WatermarkOffsets class
         ///     with the specified offsets.
         /// </summary>
         /// <param name="low">
-        ///     The offset of the earlist message in the topic/partition.
+        ///     The offset of the earlist message in the topic/partition. If 
+        ///     no messages have been written to the topic, the low watermark
```

Comment:
you mean high

Reply:
i mean low. I added this comment after looking into what actually happens in some detail. it's possible i screwed that up, but if so i was having a really bad day.